src/Conference/UpdateConference.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from "react";
+import Input from "../Input";
+import styled from "styled-components";
+import { useHistory, useParams } from "react-router-dom";
+import { updateConferenceById, getConferenceById } from "../api";
+
+const StyledConferenceForm = styled.div`
+  height: 400px;
+  padding: 20px;
+`;
+
+const StyledCardHeading = styled.h3`
+  background: #dbd8db;
+  width: 100%;
+  height: 55px;
+  font-size: 30px;
+  font-weight: bold;
+  text-align: center;
+  vertical-align: middle;
+`;
+
+const StyledCard = styled.div`
+  border-style: solid;
+  border-color: black;
+  border-radius: 25px;
+  background: #dbd8db;
+  width: 65%;
+  height: 370px;
+  transition: 0.3s;
+  :hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  }
+  vertical-align: middle;
+  margin: 0 auto;
+`;
+
+const UpdateForm = () => {
+  let history = useHistory();
+  const { id } = useParams();
+
+  const [conference, setConference] = useState({
+    name: "",
+    dateTime: "",
+    city: "",
+    description: "",
+    topic: ""
+  });
+
+  const fetchData = async conferenceId => {
+    try {
+      const response = await getConferenceById(conferenceId);
+      setConference({
+        name: response.name,
+        dateTime: response.dateTime.substring(0, response.dateTime.length - 1),
+        city: response.city,
+        description: response.description,
+        topic: response.topic
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    if (id) {
+      fetchData(id);
+    }
+  }, [id]);
+
+  const handleChange = event => {
+    const value = event.target.value;
+    setConference({ ...conference, [event.target.name]: value });
+  };
+
+  const submitForm = async () => {
+    const newConference = {
+      ...conference,
+      dateTime: conference.dateTime + "Z"
```

Comment:
This seems odd, do you really need to do it? I've not got time to look at it properly right now but if you really need to do it then I'd use a template literal.

Reply:
Unfortunately due to how date time-local outputs a date we do, or it'll be refused by the back end as it doesn't fit the Instant format

Reply:
but I agree with the template literal